src/main/java/com/spotify/dns/DnsSrvResolvers.java

```diff
@@ -78,6 +82,10 @@ public DnsSrvResolver build() {
 
       LookupFactory lookupFactory = new SimpleLookupFactory();
 
+      if (!useLookupCache) {
```

Comment:
It looks like `useLookupCache`and `cacheLookups` conflict if both are set to true. Is that the case? The builder should disallow or handle that somehow, if so.